packages/components/src/entities/models.ts

```diff
@@ -1,4 +1,36 @@
+import { ComponentType } from 'react';
+
+import { User } from '../internal/components/base/models/User';
+import { QueryModel } from '../public/QueryModel/QueryModel';
+
 export interface AssaySampleColumnProp {
     fieldKey: string;
     lookupFieldKey: string;
 }
+
+interface SampleStorageLocationComponentProps {
+    actionChangeCount?: number;
+    currentProductId?: string;
+    onUpdate?: () => void;
+    sampleId: string | number;
+    updateAllowed: boolean;
+    user: User;
+}
```

Comment:
We'll have to remember if we update the component in inventory that we need to update here as well. I wonder if a comment near the storage location component is worthwhile (and similar for other components referenced here).

Reply:
Yes, I'll add a comment to each of them.